### PR TITLE
Create info.ini fpr chance.js

### PR DESCRIPTION
**Pull request by siscia**

### CODE REVIEW

File: files/chance/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "chance",
+  "repo": "victorquinn/chancejs",
+  "files": {
+    "basePath": "dist/"
```
Comment (jimaek) [inline files/chance/update.json]: There is no dist/ directory here https://github.com/victorquinn/chancejs/tree/0.7.6
The bot will fail

**Comment by siscia** [comment]
I am not really into JS development.

The github repo does not provide a minified, how this can be fixed ?

File: files/chance/update.json (hunk 1 above)
Comment (victorquinn) [inline files/chance/update.json]: I just pushed a new version of Chance with a `dist` folder containing a minified version of Chance so this should theoretically work now!

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
Thank you all

**Pull request closed without merging**